dvc/remote/base.py

```diff
@@ -27,16 +27,13 @@
 from dvc.path_info import PathInfo, URLInfo
 from dvc.utils.http import open_url
 
-
```

Comment:
No need to remove this line and the ones down below. Please install pre-commit hooks, as required in the contribution guide https://dvc.org/doc/user-guide/contributing , so it will handle it for you. Or you can also run `black .` to fix the formatting.